packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/members/add-ip-instance/style.less

```
@@ -2,3 +2,9 @@
   max-height: 50dvh;
   overflow-y: scroll;
 }
+
+.box-member {
+  .oui-field {
+    margin-bottom: 0;
```

you can use css bootsrap 
```suggestion
   @extend .mb-0;
```

---

Extending mb-0 doesn't not work with less files, and switching to scss require to import 4 bootstrap4 files, so this will stay as is